04/js/app.js

```diff
@@ -0,0 +1,43 @@
+// 1. Funkcja z 20 eleemtami
+
+function createArr(min, max) {
+  const newArr = [];
+
+  for (let i = min; i <= min + 19; i++) {
```

Comment:
Tutaj interesuje nas ilość wykonanych operacji wiec w żaden sposób nie jest to związane z `min`, dlatego pętla będzie wyglądać tak: `for(let i=0; i<20;i++)` - tak będzie prościej :)